# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/ipc/tests/file_cross_frame.html
# Commit: 07c8392cb04d
# Full Hash: 07c8392cb04d095caa8e65ee1a10ccc5f7352eb6
# Author: Andrew McCreight <continuation@gmail.com>
# Date: 2021-05-07 21:46:25
# Regressor Bug: 1618547
# File Overlap Count: 1
# Description:
#   Bug 1618547 - Support Fission in the process priority manager. r=kmag,gsvelto
#   
#   With Fission, there can be multiple BrowserParents in a single tab, so
#   this patch moves the tracking of active tabs onto the top BrowsingContext
#   in a tab. If the priority of a top BC is changed, then the activity
# ==============================================================================

diff -r 285861b32934 -r 07c8392cb04d dom/ipc/tests/file_cross_frame.html
--- /dev/null	Thu Jan 01 00:00:00 1970 +0000
+++ b/dom/ipc/tests/file_cross_frame.html	Fri May 07 17:51:34 2021 +0000
@@ -0,0 +1,12 @@
+<!DOCTYPE HTML>
+<html>
+<head>
+  <meta charset="utf-8">
+  <title>Different-origin iframe</title>
+</head>
+<body>
+<iframe id="testIFrame" src="http://example.org/browser/dom/ipc/tests/file_dummy.html"></iframe>
+<i>I am a web page</i>
+</div>
+</body>
+</html>